### ISSUE TITLE
Documentation Issues - October-December 2016

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#5896]
- [x] For AirTerminal:SingleDuct:VAV:NoReheat. There is an inconsistency between v8.6 I/O ref (5 choices) and the IDD (3 choices) per the Zone Minimum Air Flow Input Method.  The IDD is correct and consistent with the source code.  This field description (and related fields that follow) should be identical to the same fields in AirTerminal:SingleDuct:VAV:Reheat - review both for accuracy.

**Comment by mjwitte** [comment]
- [x] In the EMS Application Guide, the description for the "Window Shading Control" actuator does not state the actuator name directly in the text as the other actuator descriptions do. 
- [x] In the EMS example "Example 6. Window Shade Control" idf listing, the actuator control type is "Window Shade Control" which should be "Window Shading Control".

**Comment by mjwitte** [comment]
- [x] Some of the advanced plant loop capacitance outputs are not documented in the I/O Ref.  The first two are there, but not the rest.

```
Output:Variable,*,Plant Demand Side Lumped Capacitance Temperature,hourly; !- HVAC Average [C]
Output:Variable,*,Plant Supply Side Lumped Capacitance Temperature,hourly; !- HVAC Average [C]
Output:Variable,*,Plant Demand Side Lumped Capacitance Heat Transport Rate,hourly; !- HVAC Average [W]
Output:Variable,*,Plant Supply Side Lumped Capacitance Heat Transport Rate,hourly; !- HVAC Average [W]
Output:Variable,*,Plant Demand Side Lumped Capacitance Heat Storage Rate,hourly; !- HVAC Average [W]
Output:Variable,*,Plant Supply Side Lumped Capacitance Heat Storage Rate,hourly; !- HVAC Average [W]
Output:Variable,*,Plant Demand Side Lumped Capacitance Excessive Storage Time,hourly; !- HVAC Sum [hr]
Output:Variable,*,Plant Supply Side Lumped Capacitance Excessive Storage Time,hourly; !- HVAC Sum [hr]
```

**Comment by mjwitte** [comment]
- [x] Add guidance about schedules in various places, including tips and tricks.  Something that makes the same point as this response regarding HVAC component schedules.

> Regarding component schedules, the general rule (which probably isn't stated anywhere in the documentation) is don't schedule anything except the supply fan and the corresponding availability manager(s).  Beyond that, everything should always be available and let the controls determine what runs or doesn't run.  This gets most confusing when using a night cycle manager - if a component schedule other than the supply fan it won't run even when night cycle turns on the system.

**Comment by mjwitte** [comment]
- [x] Add guidance in availability managers section(s) and tips and tricks like this.

> for unitary systems, don't use the night cycle manager.  Use a scheduled availability manager and let the system be always on.  then use the Supply Air Fan Operating Mode Schedule Name in the unitary system to switch between continuous fan (for ventilation) during occupied periods and switch to cycling fan for unoccupied.  The system will cycle on as the thermostat requests, and this way it will run just enough to meet the load - no need for a minimum cycle time.

**Comment by JasonGlazer** [comment]
- [x]  Add some guidance on how the ASHWAT (EquivalentLayers) can be used with the Daylighting:Controls objects (or explicitly state that it cannot).

**Comment by rraustad** [comment]
- [x] In Engineering Reference V8.5 the Fan Coil Sizing section on pg 553 is poorly written. For example, cold water volume flow rate is notated at Vmax,hw. Also, equation variables are not well defined.

**Comment by mjwitte** [comment]
- [x] I/O Ref AvailabilityManager:NightCycle, inputs section, the formatting on Field subheadings is wrong for all of the fields.

**Comment by mjwitte** [comment]
- [x] I/O Ref, WindowMaterial:Shade, numeric fields 5 and 6 in the IDD are:
  
  N5 , \field Infrared Hemispherical Emissivity
  N6 , \field Infrared Transmittance

In the I/O Ref, the names are _Thermal_ Hemispherical Emissivity and _Thermal_ Transmittance.  Not sure which name is batter (check other similar fields) but they should be the same, of course.

Nigusse added:
Changed "Infrared" to "Thermal" to make it consistent with I/O Ref and will add notes as well as needed. Here is the note:
"       \note Thermal means the long-wave radiation or sometimes it is also defined as the 
        \note the Infrared wave-length range radiation."

Changed in the following objects (both in I/O Ref and IDD):
WindowMaterial:Blind,
WindowMaterial:Glazing,
WindowMaterial:Glazing:RefractionExtinctionMethod,
WindowMaterial:Shade:EquivalentLayer, 
WindowMaterial:Drape:EquivalentLayer,
WindowMaterial:Screen:EquivalentLayer,
WindowMaterial:Glazing:EquivalentLayer,

**Comment by mjwitte** [comment]
- [x] Per JH bldg-sim post on 11/11/2016 - As the co-author of the Zhang-Huang Model, I can say that the information in your first sentence is garbled.  The ZHM calculates the global horizontal radiation based on various climatic parameters, i.e., cloud cover, temperature change over 3 hours, wind speed, etc.  It is NOT used to derive the direct/diffuse split, so I don't understand how that has gotten into the EnergyPlus documentation. I have mentioned this misunderstanding to members of the EnergyPlus Team five years ago or so, but it seems it still persists.

_I have not located the offending text yet._

**Comment by JasonGlazer** [comment]
- [x] Add all the native variables related to time of use to the list under UtilityCost:Tariff under the Time Of Use Period Schedule Name. See the list of all native variables in any Tariff report in the tabular output. For example "PeakAndShoulderEnergy" is missing along with many others.

**Comment by mjwitte** [comment]
- [x] In the IDD, for AirTerminal:SingleDuct:VAV:NoReheat and AirTerminal:SingleDuct:VAV:Reheat, the min flow fraction and min flow rate fields became autosizeable in #5646 but there are no IDD notes to explain how it will be sized.  A condensed version of this would be useful  (from the I/O Ref):
This field is autosizable and defaulted to autosize. The values for autosizing are picked up from
the Sizing:Zone input fields ”Cooling Minimum Air Flow per Zone Floor Area”, ”Cooling Minimum
Air Flow”, and ”Cooling Minimum Air Flow Fraction”. If there is no sizing calculation a default of
.000762 cubic meters per second per square meter of zone floor area (0.15 cfm/ft2) is used.

BN added:
Also added I/O Ref section to AirTerminal:SingleDuct:VAV:NoReheat. Adopted from AirTerminal:SingleDuct:VAV:Reheat

**Comment by rraustad** [comment]
- [x] The EIR chiller equation for ChillerEIRfPLR has been slightly corrupted in V8.5 and V8.6 leading to a question on UnmetHours at https://unmethours.com/question/21101/eir-chiller-performance-factors/?answer=21912#post-id-21912.

Also need to clarify this sentence describing the equation in text. We should change this sentence to read:

"The output of this curve is multiplied by the reference EIR (inverse of the reference COP) and the Energy Input to Cooling Output Ratio Function of Temperature Curve to give the energy rate (power) at the specific temperature and part-load ratio at which the chiller is operating." 

![chillerpowerdiscussion](https://cloud.githubusercontent.com/assets/5778156/21722023/d83be6f0-d3f8-11e6-9263-c66f38f66118.png)

**Comment by Nigusse** [comment]
- [x] The total power at rated conditions on page 807, equation 16.167 (E+ V8.6) in Engineering Reference is incorrect. In the EER calculation, Total Cooling Cooling Capacity Function of Temperature curve is evaluated with 'Test B' conditions but it should have been rated conditions (19.44 WB/35 DB). The source code shows that it is correctly implemented. The "(TotCapTempModFac_TestB)" term should be replaced by "(TotCapTempModFac_Rated)".  See Ticket ID: 11856.

**Comment by Nigusse** [comment]
@mjwitte
The "Zhang-Huang Solar Model" in Section 5.1.4.3 of the Eng. Ref (V8.6) or develop version does not mention direct/diffuse split. But the following paragraph in Section 5.1.5 "Perez Direct/Diffuse Splitting Model" discusses splitting global horizontal radiation. However, this two section are not related. My be the confusion is caused because the two sections are next to each other. I also checked the I/O Ref but found nothing offending text. So there is nothing to address here.

**Comment by Nigusse** [comment]
@JasonGlazer I am looking at "Add all the native variables related to time of use to the list under UtilityCost:Tariff under the Time Of Use Period Schedule Name" documentation issue. 

Looking at the I/O Ref, some of the native variables such as: IsWinter, IsNotWinter, IsSpring, IsNotSpring, IsSummer, IsNotSummer, IsAutumn, IsNotAutumn, may need to go into  "Season Schedule Name" field, I think. I am not sure if all the native variables go into "Time Of Use Period Schedule Name" input field.

**Comment by mjwitte** [comment]
@Nigusse Looking further down in the original bldg-sim message, there is a link to this section of the Auxiliary Programs Doc:

> However, many data sources may not have any solar radiation components. A study
was undertaken to find an appropriate solar model to fill in missing solar data for weather
files. The goal was to determine one or more possible solar models to use in the weather
conversion/creation process. One model seemed better overall with the usual given data
from the sources than others. The model, termed Zhang-Huang, has been used in a variety
of locations and data creations, including the upcoming IWEC2 data. The model uses a
simplistic approach of recent drybulb temperatures, sky cover, global solar constant and
solar position. This model is only used when all solar values are missing from the incoming
data. When global radiation is available, then a different model (Perez split) is used to split
the global into direct normal and diffuse horizontal values.

I would take the last sentence about Perez out of this paragraph and add it to the paragraph that comes before this, so the progression of this section would be:
a) If you have two components, then you can easily compute the third
b) If you have global horizontal then Perez is used for the direct/diffuse split.
c)  If you have no solar data, then Zhang-Huang is used to estimate global horizontal, and Perez is used to split that into direct and diffuse.

**Comment by JasonGlazer** [comment]
@Nigusse I just want to make sure all the native variables that show up in the Tariff Report are documented. If they are listed in two places, some language that leads the user that is reading either field description that other variables are described in the other field would be needed.

**Comment by Nigusse** [comment]
@mjwitte Ok. updated the  Auxiliary Programs text. Thanks.

**Comment by Nigusse** [comment]
@JasonGlazer Will follow the IO Ref document and the Tariff Report to make determination where to document the variables. Thanks.

**Comment by Nigusse** [comment]
Addressed all items listed in this issue. Waiting for the CI machine run for verification.

**Comment by Myoldmopar** [comment]
Closed via #5996
